### PR TITLE
Add getPartitionKeyRanges method and types

### PR DESCRIPTION
**Pull request by Inrixia**
RE #170

This adds the `getPartitionKeyRanges` method & arg/return types for fetching `PartitionKeyRanges`.

I have not added tests.

### CODE REVIEW

File: packages/cosmos/src/types.ts [hunk 1]
```diff
@@ -26,6 +26,22 @@ export interface Database extends PersistedResource {
...
+  id: string;
+  minInclusive: string;
+  maxExclusive: string;
+  ridPrefix: number;
+  throughputFraction: number;
+  status: string;
+  parents: unknown[];
+}
+
+export type PartitionKeyRanges = {
```
Comment (jdanyow) [inline packages/cosmos/src/types.ts:39]: I think the PartitionKeyRanges object will be of type PersistedResource
https://docs.microsoft.com/en-us/rest/api/cosmos-db/get-partition-key-ranges#body-1
```ts
export interface PartitionKeyRanges extends PersistedResource {
  PartitionKeyRanges: PartitionKeyRange[];
   _count: number;
}
```

File: packages/cosmos/src/types.ts [hunk 2]
```diff
@@ -26,6 +26,22 @@ export interface Database extends PersistedResource {
   _users: 'users/';
 }
 
+export interface PartitionKeyRange extends PersistedResource {
```
Comment (jdanyow) [inline packages/cosmos/src/types.ts:29]: I don't think this extends PersistedResource

**Review by jdanyow — CHANGES_REQUESTED** [review]
Please add an integration test in the set of collections tests: 
https://github.com/cfworker/cfworker/blob/main/packages/cosmos/test/client.integration.spec.ts#L55

Something similar to this one is all we need:
https://github.com/cfworker/cfworker/blob/main/packages/cosmos/test/client.integration.spec.ts#L103-L108

**Comment by Inrixia** [comment]
Yes sorry have been busy. Will eventually get back to this :)

**Pull request closed without merging**